cranelift/filetests/filetests/isa/riscv64/reduce.clif

```diff
@@ -10,10 +10,12 @@ block0(v0: i128):
 
 ; VCode:
 ; block0:
+;   mv a0,a0
```

Comment:
I'm hoping that whatever is causing `mv a0,a0` in the riscv backend is easy to fix.